yt/yt/client/queue_client/config.cpp

```diff
@@ -67,15 +71,30 @@ void TQueueStaticExportConfig::Register(TRegistrar registrar)
         .Default(false);
 
     registrar.Postprocessor([] (TThis* config) {
-        if (config->ExportPeriod.GetValue() % TDuration::Seconds(1).GetValue() != 0) {
-            THROW_ERROR_EXCEPTION("The value of \"export_period\" must be a multiple of 1000 (1 second)");
+        if (config->ExportPeriod) {
+            if (config->ExportPeriod->GetValue() % TDuration::Seconds(1).GetValue() != 0) {
+                THROW_ERROR_EXCEPTION("The value of \"export_period\" must be a multiple of 1000 (1 second)");
+            }
+            config->ExportSchedule = *config->ExportPeriod;
+        } else if (config->ExportCronExpression) {
+            try {
+                config->ExportSchedule.emplace<TCronExpression>(*config->ExportCronExpression);
+            } catch (const yexception& exc) {
```

Comment:
We usually just catch std::exception, our own YT exceptions have a separate path, and yexception is yandex-util-specfic, so best to be generic.